LCR link updates

tpetchel opened the pull request.

chef-delivery commented:
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

jamescott commented:
@delivery approve

chef-delivery commented:
Change: ef953723-8e39-4918-a06b-886321d9741f approved by: @jamescott

chef-delivery commented:
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

Commit:
Updated links for Learn Chef tutorials

Signed-off-by: David Wrede <dwrede@chef.io>

tpetchel committed:
Add more links

Add links for the content that was deleted in the prior commit. (We
didn’t know the links at the time.)

@davidwrede: I didn’t run Sphinx locally to verify the change - can you
spot-check my use of underscores at the end of the links - I forgot
what their purpose was. Thanks!

Commit:
Fixed dup'ed anchor text

Signed-off-by: David Wrede <dwrede@chef.io>

davidwrede reviewed (approved):
Spot checked the links and they seem to resolve correctly.
